template-compose/app/build.gradle.kts

```diff
@@ -0,0 +1,174 @@
+    defaultConfig {
+        applicationId = "co.nimblehq.template.compose"
+        minSdk = Versions.ANDROID_MIN_SDK_VERSION
+        targetSdk = Versions.ANDROID_TARGET_SDK_VERSION
+        versionCode = Versions.ANDROID_VERSION_CODE
+        versionName = Versions.ANDROID_VERSION_NAME
+    }
+
+    buildTypes {
+        getByName(BuildType.RELEASE) {
+            isMinifyEnabled = true
+            isDebuggable = false
+            isShrinkResources = true
+            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+            signingConfig = signingConfigs[BuildType.RELEASE]
+            buildConfigField("String", "BASE_API_URL", "\"https://jsonplaceholder.typicode.com/\"")
+        }
+
+        getByName(BuildType.DEBUG) {
+            // For quickly testing build with proguard, enable this
+            isMinifyEnabled = false
+            signingConfig = signingConfigs[BuildType.DEBUG]
+            buildConfigField("String", "BASE_API_URL", "\"https://jsonplaceholder.typicode.com/\"")
+        }
+    }
+
+    flavorDimensions(Flavor.DIMENSIONS)
+    productFlavors {
+        create(Flavor.STAGING) {
+            applicationIdSuffix = ".staging"
+        }
+
+        create(Flavor.PRODUCTION) {}
+    }
+
+    sourceSets["test"].resources {
+        srcDir("src/test/resources")
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_11.toString()
+    }
+
+    composeOptions {
+        kotlinCompilerVersion = Versions.KOTLIN_VERSION
+        kotlinCompilerExtensionVersion = Versions.COMPOSE_COMPILER_VERSION
+    }
+
+    buildFeatures {
+        viewBinding = true
+        compose = true
+    }
+
+    lintOptions {
+        isCheckDependencies = true
+        xmlReport = true
+        xmlOutput = file("build/reports/lint/lint-result.xml")
+    }
+
+    testOptions {
+        unitTests {
+            // Robolectric resource processing/loading
+            isIncludeAndroidResources = true
+            isReturnDefaultValues = true
+        }
+        unitTests.all {
+            if (it.name != "testStagingDebugUnitTest") {
+                it.extensions.configure(kotlinx.kover.api.KoverTaskExtension::class) {
+                    isDisabled.set(true)
+                }
+            }
+        }
+    }
+}
+
+kapt {
+    correctErrorTypes = true
+}
+
+dependencies {
+    implementation(project(Module.DATA))
+    implementation(project(Module.DOMAIN))
+
+    implementation(fileTree(mapOf("dir" to "libs", "include" to listOf("*.jar"))))
+
+    implementation("androidx.activity:activity-ktx:${Versions.ANDROIDX_ACTIVITY_KTX_VERSION}")
+    implementation("androidx.appcompat:appcompat:${Versions.ANDROIDX_SUPPORT_VERSION}")
+    implementation("androidx.constraintlayout:constraintlayout:${Versions.CONSTRAINT_LAYOUT_VERSION}")
```

Comment:
We can remove these `libs` for `template-compose` as we don't require it anymore 🙏

Reply:
Code removal should be done in #334 🙏

Reply:
@Wadeewee Done in #351 🙏🏼